pkg/model/proxy.go

```diff
@@ -243,6 +243,7 @@ type NodeMetadata struct {
 	MeshID string `json:"MESH_ID,omitempty"`
 
 	// ClusterID defines the cluster the node belongs to.
+	// This is sent as an unverified metadata in gRPC requests. Ignored if PILOT_CENTRAL_ISTIOD is false.
```

Comment:
Right now we have proxy.VerifiedIdentity which is the bits of the identity that have been actually been verified vs just what they told us. Do you think it makes sense to have similar here?